src/reservations/reservations.service.spec.ts

```
@@ -48,12 +49,21 @@
       ),
     };
 
+    const mockFilesService = {
+      getUploadPresignedUrl: jest.fn().mockResolvedValue({
+        presignedUrl: 'https://example.com/upload',
+        filePath: 'test/file.jpg',
+      }),
+      getAccessPresignedUrl: jest.fn().mockResolvedValue('https://example.com/access'),
```

ESLint

[error] Replace `.fn()` with `⏎········.fn()⏎········`

(prettier/prettier)